test_crates/module_missing/old/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+mod a {
+}
+
+mod b {
+    pub mod b {
+    }
```

Comment:
Any reason not to just do `{}` on the same line throughout when the module is empty?